tests/output/pjs/ast_transform_test.js

```diff
@@ -1,3 +1,17 @@
+import PJSCodeInjector from "../../../js/output/pjs/pjs-code-injector.js";
+
+const getCodeFromOptions = function(code) {
```

Comment:
Where is this used?  Did you maybe forget an `export`?

Reply:
Great catch. It's used a bunch later in the file. I'm changing it to import from pjs-utils instead. I'm confused as to why I pasted a copy in the file instead.